src/main/java/decodes/dcpmon/XRWriteThread.java

```diff
@@ -77,6 +77,7 @@ class XRWrapper
 
 	public XRWriteThread()
 	{
+		super("XRWriteThread");
```

Comment:
For debugging would we need a specific thread name for each Instance?

Reply:
For this particular component there's only ever one instance. But having a name anyways helps with sorting through threads.